packages/checkout-ui-extensions/documentation/components.md

```diff
@@ -68,7 +67,7 @@ Checkout components are designed to be flexible, enabling you to layer and mix t
     <h3 class="checkout-component-grid-item_title">Other</h3>
       <p><code><a href="../src/components/Button">Button</a></code></p>
       <p><code><a href="../src/components/Link">Link</a></code></p>
-      <p><code><a href="../src/components/List">List</a></code></p>
+      <p><code><a href="../src/components/List">List</a></code> &amp; <code><a href="../src/components/ListItem">ListItem</a></code></p>
```

Comment:
Is this what's going to show on the overview page?
I think dev doesn't doesn't use ampersand. I think they prefer "and".

Reply:
I have a preference, but I dont mind either way. Pick one :)

Reply:
oh that's if we want grouping. I'm good with or without, will let you two decide that (Owen knows my eye for design is ... weak)